src/createServer.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { router: userRouter } = require('./routes/user.route.js');
+const { router: expenseRouter } = require('./routes/expense.route.js');
+const { resetUsers } = require('./services/user.service.js');
+const { resetExpenses } = require('./services/expense.service.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
```

Comment:
Resetting users and expenses data on server startup can lead to data loss in a production environment. Consider moving these calls to a development-only environment or a separate initialization script.